data/experiment-recipe-samples/rollout-mobile.json

```diff
@@ -0,0 +1,38 @@
+{
+  "schemaVersion": "0.1.0",
+  "id": "rollout-mobile-example",
+  "slug": "rollout-mobile-example",
+  "application": "reference_browser",
+  "appId": "org.mozilla.reference.browser",
+  "appName": "reference_browser",
+  "featureIds": ["bookmark-icon"],
+  "channel": "nightly",
+  "userFacingName": "Mobile Rollout Example",
+  "userFacingDescription": "An A/A Test to validate rollout in the Rust SDK",
+  "isRollout": true,
+  "isEnrollmentPaused": false,
+  "bucketConfig": {
+    "randomizationUnit": "nimbus_id",
+    "namespace": "rollout-mobile-example-rollout-1",
+    "start": 0,
+    "count": 5000,
```

Comment:
We would normally test with a rollout created for 100% of the population, as it is the most likely scenario.